src/Factory/RestControllerFactory.php

```diff
@@ -143,7 +143,6 @@ public function createServiceWithName(ServiceLocatorInterface $controllers, $nam
             ));
         }
 
-        $controller->setEventManager($events);
```

Comment:
This line caused a major BC break for us, our app stopped working. Controller events like this are not fired anymore:
```$events->attach('ZF\Rest\RestController', 'fetch', array($this, 'onFetch'),10);```